### DISCUSSION TITLE
How to show results of a static model using OpenModelica?

### DISCUSSION THREAD

**Discussion by s-celles** [OpenModelica/OpenModelica#11345] [Q&A] [answered]
Hello,

I'd like to display results of a static model (ie not time varying).

I noticed this SO post https://stackoverflow.com/questions/76359121/how-to-show-results-of-a-static-model-in-modeling-view-with-openmodelica which suggests using `Modelica.Blocks.Interaction.Show.RealValue`.

Unfortunately I haven't found "Diagram Window" of the plotting view which is mentioned in an answer.

Any idea where I can find it ?

Kind regards

**Comment by adrpo** [comment, marked as answer]
You simulate the model, it will get you into the Plotting perspective, then you click on the OMEdit looking button 
![image](https://github.com/OpenModelica/OpenModelica/assets/4192979/bf78b24b-5d60-49eb-880d-9dd553e47d35)
 on the toolbar to show the diagram that can be animated in the Plotting perspective. See here:
https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/omedit.html#omedit-diagram-window
